modules/custom/openy_addthis/config/install/openy_addthis.settings.yml

```diff
@@ -0,0 +1,3 @@
+public_id: ra-59662efe3f09cfcc
```

Comment:
1. This setting should be moved to the installation step where we have all API keys 3rd party services https://github.com/ymcatwincities/openy/blob/8.x-1.x/src/Form/ThirdPartyServicesForm.php.
2. The demo value shouldn't belong to the module. It should be part of the demo content. Please add separate module to the https://github.com/ymcatwincities/openy/tree/8.x-1.x/modules/openy_features/openy_demo_content/modules package and enable it during installation https://github.com/ymcatwincities/openy/blob/8.x-1.x/openy.profile#L49.